src/ncp/ncp_base.cpp

```diff
@@ -1545,7 +1565,7 @@ ThreadError NcpBase::CommandHandler_PROP_VALUE_GET(uint8_t header, unsigned int
     spinel_ssize_t parsedLength;
     ThreadError errorCode = kThreadError_None;
 
-    parsedLength = spinel_datatype_unpack(arg_ptr, arg_len, "i", &propKey);
+    parsedLength = spinel_datatype_unpack(arg_ptr, arg_len, SPINEL_DATATYPE_UINT_PACKED_S, &propKey);
```

Comment:
here its a propKey.  I could settle for that name if it was used consistently throughout.